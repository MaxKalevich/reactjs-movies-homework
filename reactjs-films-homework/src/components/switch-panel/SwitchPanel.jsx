import styles from "./switch-panel-style.module.scss";

const SwitchPanel = () => {
  return (
    <div className={styles.switchWrapper}>
      <ul className={styles.itemList}>
        <li>Popular</li>
        <li>Top Rated</li>
        <li>Upcoming</li>
      </ul>
    </div>
  );
};

export default SwitchPanel;
