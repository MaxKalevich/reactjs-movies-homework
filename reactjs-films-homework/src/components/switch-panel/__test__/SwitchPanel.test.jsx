import { render, screen } from "@testing-library/react";
import SwitchPanel from "../SwitchPanel";

describe("Checking LanguageButton component", () => {
  it("Contains displayText in LanguageButton component", () => {
    const { asFragment } = render(<SwitchPanel />);
    expect(asFragment(<SwitchPanel />)).toMatchSnapshot();
  });
});
