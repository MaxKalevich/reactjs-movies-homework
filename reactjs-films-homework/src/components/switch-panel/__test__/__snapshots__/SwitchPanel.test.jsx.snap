// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking LanguageButton component Contains displayText in LanguageButton component 1`] = `
<DocumentFragment>
  <div
    class="switchWrapper"
  >
    <ul
      class="itemList"
    >
      <li>
        Popular
      </li>
      <li>
        Top Rated
      </li>
      <li>
        Upcoming
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
